### DISCUSSION TITLE
Preferred level of logging for logs regarding various bugs.

### DISCUSSION THREAD

**Discussion by 6ab18b** [lightspark/lightspark#991] [Q&A] [answered]
If I were to go through various flash files and log every bug I find then roughly organize them in an archive by type and similarity and post them in another thread, **should I always do 4 or would this variate based upon what type of bug it is?** For point of reference, my folder structure would be: 
1. placing them initially by whether it's a crash or another error 
2. categorizing them by some technical aspect regarding the flash if deemed necessary 
3. whether the issue is regarding some audio, visual or interactable component
4. finally, by similarity of the issue.

**Comment by huwdp** [comment]
We have a page here (https://github.com/lightspark/lightspark/wiki/Status-of-Lightspark-support) that logs games we have tested. The page and the spreadsheet linked in there hasn't been updated in a while but any help would be appreciated. If you were to report any issues then -l 4 would be best. -l 4 may matter less for visual and audio problems but it depends on the severity of the bug

**Comment by eiglow** [comment, marked as answer]
The main thing is that the bug is described, reproduction steps are given, and the swf is attached. I don't think the logs are too important unless you can't easily reproduce the bug while running the swf in standalone lightspark.
